flexget/plugins/filter/crossmatch.py

```diff
@@ -43,6 +43,10 @@ def on_task_filter(self, task, config):
         action = config['action']
         all_fields = config['all_fields']
 
+        if len(task.entries) == 0:
```

Comment:
Seems a reasonable optimization. The more pythonic way to put this would be:
```suggestion
        if not task.entries:
```

Reply:
Yes that is a better way! Do I need to change the commit? Or is it done?

Thank you

Reply:
I merged it in, thanks!